test/fw/ptl/utils/pbs_logutils.py

```diff
@@ -279,6 +274,31 @@ def convert_date_time(cls, dt=None, fmt=None, syslog=False):
         else:
             return int(tm)
 
+    def get_syslog_date_str(self, logline=None):
+        if logline is None:
+            return None
+
+        if "T" in logline:
+            return logline[:19]
+        else:
+            return logline[:15]
+
+    def convert_syslog_time(self, dt=None):
+        if dt is None:
+            return None
+
+        if 'T' in dt:
```

Comment:
can you please add a comment to explain what the input format is and what format you want to convert it to (the output)? an example might be sufficient.